src/Stryker.Core/Stryker.Core/Mutators/Mutator.cs

```diff
@@ -42,7 +42,9 @@ public enum Mutator
         [MutatorDescription("Null coalescing")]
         NullCoalescing,
         [MutatorDescription("Math methods")]
-        Math
+        Math,
+        [MutatorDescription("Pattern matching expressions")]
+        PatternMatching
```

Comment:
This is not a new mutator type. They fall under Logical and Equality operators

Reply:
Done